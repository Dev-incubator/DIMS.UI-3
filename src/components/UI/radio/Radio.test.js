import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Radio from './Radio';

configure({
  adapter: new Adapter(),
});

describe('<Radio />', () => {
  it('Snapshot Radio element', () => {
    const wrapper = shallow(<Radio />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
