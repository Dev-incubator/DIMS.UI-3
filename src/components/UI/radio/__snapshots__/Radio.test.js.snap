// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio /> Snapshot Radio element 1`] = `
<label>
  <input
    className="radio"
    name="base"
    type="radio"
  />
</label>
`;
